### ISSUE TITLE
What if we train CNN to Vertex data?

### ISSUE THREAD

**Issue by FilipsFedotovs** [FilipsFedotovs/EDER-VIANN#61]
Create data with CNN training performance. (Validation Loss, Test loss, Accuracy etc. vs Epochs)
